app/Http/Backend/Controllers/CategoryController.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Http\Backend\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\Http\Backend\Requests\AddCategoryRequest;
+use App\Http\Backend\Requests\EditCategoryRequest;
+use App\Category;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $data = Category::select('id', 'name', 'image')->orderBy('id', 'DESC')->get();
+        return view('backend.category.list', compact('data'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('backend.category.add');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CategoryRequest $request)
```

Comment:
line 8 `use App\Http\Backend\Requests\AddCategoryRequest;`
But why in this function you inject `CategoryRequest`